index.js

```diff
@@ -2,66 +2,74 @@
 'use strict';
 
 var path = require('path');
+var fs = require('fs');
 
 var EventEmitter = require('events').EventEmitter;
 var mergeTrees = require('broccoli-merge-trees');
-var VersionChecker = require('ember-cli-version-checker');
 var FastBootExpressMiddleware = require('fastboot-express-middleware');
 var FastBoot = require('fastboot');
 var chalk = require('chalk');
 
-var patchEmberApp     = require('./lib/ext/patch-ember-app');
 var fastbootAppModule = require('./lib/utilities/fastboot-app-module');
+var FastBootConfig = require('./lib/broccoli/fastboot-config');
+var migrateInitializers = require('./lib/build-utilities/migrate-initializers');
+var fastbootBuilder = require('./lib/build-utilities/fastboot-builder');
 
-var filterInitializers = require('fastboot-filter-initializers');
-var FastBootBuild      = require('./lib/broccoli/fastboot-build');
+const Concat = require('broccoli-concat');
+const p = require('ember-cli-preprocess-registry/preprocessors');
+const existsSync = require('exists-sync');
 
 /*
  * Main entrypoint for the Ember CLI addon.
  */
-
 module.exports = {
   name: 'ember-cli-fastboot',
 
+
   init() {
     this._super.init && this._super.init.apply(this, arguments);
+    // set a environment variable to allow addons to use `fastboot-filter-initializers`
+    // for old versions.
+    process.env.FASTBOOT_NEW_BUILD = true;
```

Comment:
I wanted this variable to be set early on. Isn't `included` called after `init`?